### PR TITLE
Update custom app org switch in teams-custom-app-policies-and-settings.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/9329.

### CODE REVIEW

File: Teams/teams-custom-app-policies-and-settings.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ If this setting is turned on:
 
 ### Org-wide custom app setting
 
-The **Allow interaction with custom apps** org-wide custom app setting on the [Manage apps](manage-apps.md) page applies to everyone in your organization and governs whether they can upload or interact with custom apps. This setting acts as a master on/off switch for the user and team custom app policy settings. It's intended to serve as a master on/off switch during security events. As such, user and team custom app policy settings will not take effect unless the org-wide custom app setting is enabled even if user and team custom app policy settings are enabled.
+The **Allow interaction with custom apps** org-wide custom app setting on the [Manage apps](manage-apps.md) page applies to everyone in your organization and governs whether they can upload or interact with custom apps. This setting overrides the user and team custom app policy settings. It's intended to serve as a master on/off switch during security events. If this setting is set to Off, only a Teams Service Admin or a Global admin can upload custom apps and users will be able to interact with them.
```
Comment (JohanFreelancer9) [inline Teams/teams-custom-app-policies-and-settings.md:97]:
```suggestion
The **Allow interaction with custom apps** org-wide custom app setting on the [Manage apps](manage-apps.md) page applies to everyone in your organization and governs whether they can upload or interact with custom apps. This setting overrides the user and team custom app policy settings. It's intended to serve as a master on/off switch during security events. If this setting is set to Off, only a Teams Service admin or a Global admin can upload custom apps and users will be able to interact with them.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ashishguptaiitb PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ashishguptaiitb** [comment]
Folks, thanks for coordinating this. I shall have to review this internally with the PMs before acting on the bug or this PR.

**Comment by ashishguptaiitb** [comment]
Closing this old PR. The updated and accurate info was updated in the [same help article](https://learn.microsoft.com/en-us/microsoftteams/teams-custom-app-policies-and-settings).

**Pull request closed without merging**